### ISSUE TITLE
aci_l3_domain_profile and aci_physical_domain returning: Error: unknown attribute 'descr' in element 'physDomP'

### ISSUE THREAD

**Issue by 020003** [CiscoDevNet/terraform-provider-aci#149]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
Terraform v0.13.5

### APIC version and APIC Platform
V 5.0(1h) on-prem
V 5.1 on-prem
* V x.x.x and on-prem/cloud-aws/cloud-azure/all.

### Affected Resource(s)

aci_l3_domain_profile
aci_physical_domain
### Terraform Configuration Files

```hcl
resource "aci_physical_domain" "physDomains" {
  count                       = length(var.physDomains)
  name                        = var.physDomains[count.index].name
  relation_infra_rs_vlan_ns   = aci_vlan_pool.physVlanPools[count.index].id
}


#### Create L3 Domains

resource "aci_l3_domain_profile" "l3Domains" {
  count                       = length(var.l3Domains)
  name                        = var.l3Domains[count.index].name
  relation_infra_rs_vlan_ns   = aci_vlan_pool.l3VlanPools[count.index].id
}

```

### Expected Behavior

Creation of some physical Domains and l3 domains should have happened.

### Actual Behavior

Failure of ressources with error
Error: unknown attribute 'descr' in element 'physDomP' on APIC 5.1 on APIC 5.0 it worked
You can see that its referring to a description field which is not existing under that class.

module.vlanPools.aci_l3_domain_profile.l3Domains[1] will be updated in-place
  ~ resource "aci_l3_domain_profile" "l3Domains" {
      + annotation                = "orchestrator:terraform"
        description               = jsonencode({})
        id                        = "uni/l3dom-multipodL3Out_RoutedDomain"
        name                      = "multipodL3Out_RoutedDomain"
        relation_extnw_rs_out     = []
        relation_infra_rs_vlan_ns = "uni/infra/vlanns-[L3_VLP]-static"
    }

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by nkatarmal-crest** [comment]
@020003  This issue has been fixed in the v0.5.1 of the provider. Please verify it once.

**Comment by 020003** [comment]
@nkatarmal-crest verified it and its working now. Thx for the quick support
